### ISSUE TITLE
Are all the pictures shown from the test set?

### ISSUE THREAD

**Issue by parap1uie-s** [liznerski/fcdd#1]
Thanks for the great work!

However, I am confused about data splitting, which is shown with anomalous heatmap. Are they all from the validation or test set? Or from the train set?

Thanks in advance.

**Comment by liznerski** [comment]
Thanks for your interest!
All anomaly heatmaps in the paper are for samples from the test or validation set. The only exception is Figure 9 in the Appendix, which displays manipulated training samples for the "background focus" experiments. For CIFAR-10, Fashion-MNIST, and MVTec-AD the test set is used (because there is no validation-test split). For ImageNet and Pascal-VOC validation data is used.

**Comment by parap1uie-s** [comment]
Thank you for such a detailed answer! :-)
